### ISSUE TITLE
autoDJ fixed playback time

### ISSUE THREAD

**Issue by JAAlperin** [mixxxdj/mixxx#12562]
### Feature Description

In the macOS djay application automix feature you can set a preference for a fixed track play time before transition to the next track. Dance studios use this to reduce fatigue of dancers and avoid their leaving the dance floor.  Somewhere between 2:30 and 3:00 seems to work best with a 7-10 second transition to the next track (usually a different dance genre).  This feature saves the user the considerable trouble of creating special shorter versions of tracks or setting outro start for every track in the automix/autoDJ playlist.

**Comment by RafaelFrance** [comment]
I think this is already done.
In Auto DJ, setting the transition duration to a negative value. Example -3 for 3 seconds of wait (silent) before to start next track.

**Comment by ronso0** [comment]
Yes, but the missing part is to start the transition (pause) at a fixed point in track / after track start point.
